install-selenium

From: Bryan Larsen <bryan@larsen.st>

# Integration Testing

It's not a real application without tests.  We're going to use
[Selenium](http://seleniumhq.org/) to do some integration testing.

The easiest way to use selenium with rails is to use `capybara`.   We'll
also need database_cleaner because capybara/selenium does not support
transactional fixtures.

Add to your Gemfile:

    group :development, :test do
      gem 'capybara', :git => 'git://github.com/jnicklas/capybara.git'
      gem 'database_cleaner;
    end
---
 Gemfile |    6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/Gemfile b/Gemfile
index 42648b1..eb810d2 100644
--- a/Gemfile
+++ b/Gemfile
@@ -51,3 +51,9 @@ gem "acts_as_list"
 gem "bluecloth"
 gem "shoulda", :group => :test
 gem "factory_girl_rails", :group => :test
+
+group :development, :test do
+  gem 'capybara', :git => 'git://github.com/jnicklas/capybara.git'
+  gem 'database_cleaner'
+end
+
